.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
This file was generated by Storybook as part of the upgrade. Due to an incompatibility between npm versions 7 and 8, upgrade requires that the `legacy-peer-deps` flag is included when command runs.